packages/jupyter/src/providers.tsx

```diff
@@ -14,10 +14,58 @@ import { useNotebookBase, useThebeConfig, useThebeCore, ThebeServerProvider } fr
 import type { Root } from 'mdast';
 import { useComputeOptions } from '@myst-theme/providers';
 
+function getThebeOptions(): CoreOptions {
+  const { thebe, binderUrl } = useComputeOptions();
+  const {
+    mathjaxUrl,
+    mathjaxConfig,
+    binder,
+    server,
+    kernelName,
+    sessionName,
+    disableSessionSaving,
+    local,
+  } = thebe;
```

Comment:
This isn't using the `lite` option, i.e. `useJupyterLite` from before. Not sure where that comes in?